settings.py

```diff
@@ -26,13 +28,16 @@
 CANVAS_SDK_DB_HOST = os.getenv("CANVAS_SDK_DB_HOST", "home-app-db")
 CANVAS_SDK_DB_PORT = os.getenv("CANVAS_SDK_DB_PORT", "5432")
 
-DATABASES = {
-    "default": {
+if dburl := os.getenv("DATABASE_URL"):
```

Comment:
This environment variable will have the values for the connection URL that home-app uses. We will need to replace the username and password with the read-only credentials.

Reply:
Oh, that's right.

Reply:
Changed.